docs/utilities/idempotency.md

```diff
@@ -100,8 +100,8 @@ If you're not [changing the default configuration for the DynamoDB persistence l
 
 ???+ info "Info: DynamoDB"
     Each function invocation will generally make 2 requests to DynamoDB. If the
-    result returned by your Lambda is less than 1kb, you can expect 2 WCUs per invocation. For retried invocations, you will
-    see 1WCU and 1RCU. Review the [DynamoDB pricing documentation](https://aws.amazon.com/dynamodb/pricing/){target="_blank"} to
+    result returned by your Lambda is less than 1kb, you can expect 2 WCUs per invocation.
+    Review the [DynamoDB pricing documentation](https://aws.amazon.com/dynamodb/pricing/){target="_blank"} to
     estimate the cost.
 
```

Comment:
We need to improve the documentation to show the new behavior, and we can even add an X-Ray image to show how the client will see it when the tracker is enabled.

But I can do it and we review it, what do you think?

![image](https://github.com/aws-powertools/powertools-lambda-python/assets/4295173/45685ded-f8c8-47c2-8939-7bf347d27546)

Reply:
Yes please do!